CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing to the Openshift Builder
+
+The OpenShift Builder is the component that drives [OpenShift Builds](https://docs.okd.io/latest/dev_guide/builds/index.html),
```

Comment:
`The OpenShift Builder is the component that drives OpenShift Builds`

nit: This sentence sounds/reads weird to me, but I don't have a good restructure for it.